tests/processes/LogPrinterThreadTest.py

```diff
@@ -2,14 +2,10 @@
 import unittest
 
 from coala_utils.ContextManagers import retrieve_stdout
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.processes.LogPrinterThread import LogPrinterThread
 
 
-class TestPrinter(LogPrinter):
-
-    def __init__(self):
-        LogPrinter.__init__(self, self)
+class TestPrinter():
```

Comment:
There's no harm in providing extra flexibility, and its not like we are giving at as trade off to some other functionality. It just gives more ways to capture data to developers.